### PR TITLE
fix(travis-ci): fix Webgl rendering test fails on Travis CI

### PR DESCRIPTION
**Pull request by jongmoon**
## Issue
  1. #225 
  2. refactoring codes.
  3. make dev dependency as latest.

####  1. set the premultipliedAlpha as false to get webgl context

```js
var gl = canvas.getContext('webgl', {
  premultipliedAlpha: false //<----Important
});
```
  - Ref: https://stackoverflow.com/a/48127936/6208362
  
####  2. Do not pass `--disable-gpu` which is passed as ChromeHeadless default setting.
  - instead, specify following options based on Chrome setting.

```
"--headless",
"--hide-scrollbars",
"--mute-audio",
"--no-sandbox",
"--disable-setuid-sandbox",
// Without a remote debugging port, Google Chrome exits immediately.
"--remote-debugging-port=9222",
```
> Although --use-gl=osmesa is not specified. it need to install libosmesa in TRAVIS setting to test sucessfully

>

### CODE REVIEW

**Comment by jongmoon** [comment]
To make this pr, I've tested it on #224 pr

File: test/unit/PanoViewer/PanoViewer.spec.js [hunk 1]
```diff
@@ -734,16 +721,15 @@ describe("PanoViewer", function() {
...
 		IT("should return instance of PanoViewer when setter is called.", () => {
 			// Given
-			let target;
-			let panoViewer;
-			let returnValues = [];
+			const returnValues = [];
+			const target = sandbox();
 
-			target = sandbox();
 			target.innerHTML = `<div"></div>`;
```
Comment (younkue) [inline test/unit/PanoViewer/PanoViewer.spec.js]: check markup

**Comment by younkue** [comment]
LGTM

File: test/unit/PanoViewer/PanoViewer.spec.js (hunk 1 above)
Comment (jongmoon) [reply test/unit/PanoViewer/PanoViewer.spec.js]: @younkue I got it!

**Pull request merged**